CONTRIBUTING.md

```diff
@@ -1 +1,148 @@
 # Contributing
+
+First of all, make sure you have all settings up for Yoga to run it locally:
+
+1. Fork and clone this repo
+2. Install dependencies using:
+
+```
+$ yarn
+```
+
+This project follows the
+[conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/) pattern
+and, to help you with it, we use
+[commitizen](http://commitizen.github.io/cz-cli/). So, everytime you want to
+commit some change, please use:
+
+```
+$ yarn commit
+```
+
+## New Component
+
+At Yoga, a Component isn't just one file. It has its own docs, tests, exports,
+native, web, and so on. To make it easy, there is a script at the root:
+
+```
+$ yarn new NewComponent
+```
+
```

Comment:
Should we mention the component theme here?

Reply:
Sure!